src/toplevel/learnocaml_toplevel_output.ml

```diff
@@ -180,6 +180,46 @@ let output_html ?phrase output html =
   Js_utils.Manip.appendChild output.container div ;
   insert output ?phrase (Html (html, div)) div
 
+
+(* Module to generate new id *)
+module Id_generator : sig
```

Comment:
The following module introduces a general-purpose: it should probably be moved in an existing utility module.

Reply:
I'm not sure this module introduces a general-purpose. Indeed, this id generation should only be use by the learnocaml-toplevel-output module to avoid consuming too many ids.
Maybe, to be clearer, I can replace the module by a closure as follows:
```ocaml
let get_fresh_id =
  let r = ref 0 in
  fun () -> r := !r +1 ; !r
```